package.json

```diff
@@ -9,7 +9,11 @@
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
-	"engines": {
+    "contributors": [{
+      "name": "Moos",
+      "email": "mooster@42at.com"
+    }],
```

Comment:
Thanks, but I don't use this field. It's annoying having to keep it updated. The commit history speaks for itself.